### DISCUSSION TITLE
Winpilot at startup

### DISCUSSION THREAD

**Discussion by fubo73** [builtbybel/Bloatynosy#208] [Q&A]
Hi, are changes applied to each Win11 start? If not how to do?
Everytime I start Winpilot, it seems all disabled.

**Comment by Belim** [comment]
In the setup, the configuration is read and applied live. If individual options deactivate after you have previously activated them, this may mean that you do not have sufficient rights to set the settings on the system.

**Comment by fubo73** [comment]
When I start it, it is presented like this:

![image](https://github.com/builtbybel/Winpilot/assets/49477802/a721c921-0c7e-4781-adc0-f4090ecf1c5b)

Then I set defaults:

![image](https://github.com/builtbybel/Winpilot/assets/49477802/6f64b533-c52a-4c04-9962-96ae82ae187e)

I close it. I open it and find the same as first image. How am I sure that settings where applied?

**Reply by Belim** [reply]
Are you running the ap in an administrative context? The Clippy bubble should display an error when trying to set settings that require elevated permissions, and you don't have them

**Comment by fubo73** [comment]
The ap launches in admin mode. Now I found that if I expand the first image I find the settings I set.
So it seems a matter of visual understanding. From my humble point of view, seeing first image sounds like no settings is applied.
Regarding second image, I understand that Ads are enabled. I need to move switch if I want to disable. Those are not "Disable Ads" switches, but "Set Ads" switches.

**Comment by fubo73** [comment]
I confirm that changes are applied at startup. It was my mistake in intepreting graphic. Thanks for supporting!
